src/common/definitions.js

```diff
@@ -112,16 +118,20 @@ const pciuAddress = (() => {
       },
       state: {
         type: 'string',
-        enum: constants.states.USA
-          .concat(constants.statesOnlyInPCIU)
-          .map(state => state.value)
+        enum: usaStates
       }
     }
   }, {
     properties: {
       country: {
+        not: {
+          type: 'string',
+          enum: [USA]
```

Comment:
This is good as is, one idea is whether we should we make the constant an array that includes 'USA', perhaps named USAOnly or somesuch, so that it is consistent with how pciuCountries is used.

Reply:
You're right, I had been using this in its non-array form elsewhere but that's not true anymore.